md2notion/NotionPyRenderer.py

```diff
@@ -324,3 +331,40 @@ def render_image(self, token):
             'source': token.src,
             'caption': alt
         }
+
+    class __HTMLParser(HTMLParser):
+
+        def __init__(self):
+            super().__init__()
+            self.result = []
+
+        def handle_starttag(self, tag, attrs):
+            if tag != "img": 
+                return
+            src = next((value for key, value in attrs if key == "src"), "")
+            alt = next((value for key, value in attrs if key == "alt"), "No caption")
```

Comment:
Preferably this would return `None` so that no caption is even uploaded to Notion